docs/dev/deploying.rst

```diff
@@ -0,0 +1,47 @@
+Deploying to production
+=======================
+
+Deploying a new site
+--------------------
+
+To deploy a new site, first make sure you know:
+
+#. the hostname the chapter prefers
+#. the chapter's contact email address
+#. the group's name (e.g. "MIT ESP" or "Yale Splash")
+#. the chapter's preferred theme
+
+Then ssh to ``diogenes``, grab the latest version of
+``esp/useful_scripts/server_setup/new_site.sh``, ``cd`` to ``/lu/sites``, and
```

Comment:
It looks like you also make it so that you can clone the entire repository into /lu/sites/directory-name and then run the script from there. Is it worth mentioning this here?